3-enrich/scala-hadoop-bad-rows/project/BuildSettings.scala

```diff
@@ -18,7 +18,7 @@ object BuildSettings {
   // Basic settings for our app
   lazy val basicSettings = Seq[Setting[_]](
     organization  := "com.snowplowanalytics",
-    version       := "0.1.0",
+    version       := "0.1.0-rc1",
```

Comment:
Fair enough - I originally went with 0.1.0 thinking of this as a new project rather than a continuation of Scala Hadoop Bad Rows. Will update to 0.2.0...